assets/js/common_utils.js

```diff
@@ -123,27 +123,27 @@ export const getTime = () => {
 
 const KNOWN_COMPATIBLE_ORT_VERSION = {
     "stable-diffusion-1.5": {
-        dev: "1.21.0-dev.20241122-a2ba3cb547",
+        dev: "1.22.0-dev.20250325-afaf4a5e63",
         stable: "",
         test: "",
     },
     "sd-turbo": {
-        dev: "1.21.0-dev.20241122-a2ba3cb547",
+        dev: "1.22.0-dev.20250325-afaf4a5e63",
         stable: "",
         test: "",
     },
     "segment-anything": {
-        dev: "1.21.0-dev.20241122-a2ba3cb547",
+        dev: "1.22.0-dev.20250325-afaf4a5e63",
         stable: "",
         test: "",
     },
     "whisper-base": {
-        dev: "1.21.0-dev.20241122-a2ba3cb547",
+        dev: "1.22.0-dev.20250325-afaf4a5e63",
         stable: "",
         test: "",
     },
     "phi-3-mini": {
-        dev: "1.21.0-dev.20241122-a2ba3cb547",
+        dev: "1.22.0-dev.20250325-afaf4a5e63",
```

Comment:
Looks like all demos use the same dev version, why not use a common `const` variable to store this version info? :)

Reply:
Now they use the same version, but can't say always the same, e.g. SA uses customized version before.

Reply:
I meant we can have a common dev version, and if some demo needs specific version, leave it as the specific version.

e.g. 

```
const ORT_DEV_VERSION = "1.22.0-dev.20250325-afaf4a5e63";
...
    "stable-diffusion-1.5": {
        dev: ORT_DEV_VERSION,
        stable: "",
        test: "",
    },
    "sd-turbo": {
        dev: "1.21.0-dev.20241122-a2ba3cb547",
        stable: "",
        test: "",
    },
```

Reply:
@Honry done, thanks for the great suggestion. PTAL